docs/advanced-features/athena.ipynb

```diff
@@ -0,0 +1,1035 @@
+    "        Name=ATHENA_WORKGROUP, Description=\"Quilt uses this for Athena SQL Queries\"\n",
+    "    )\n",
+    "    stat(wg_create)\n",
+    "\n",
+    "# Configure Workgroup to use that Bucket\n",
+    "wg_update = ATHENA.update_work_group(\n",
+    "    WorkGroup=ATHENA_WORKGROUP,\n",
+    "    ConfigurationUpdates={\n",
+    "        \"ResultConfigurationUpdates\": {\n",
+    "            \"OutputLocation\": ATHENA_URL,\n",
+    "        },\n",
+    "    },\n",
+    ")\n",
+    "stat(wg_update, \"wg_update\")\n",
+    "\n",
+    "# Create new GLUE Database\n",
+    "\n",
+    "db_create = ATHENA.start_query_execution(\n",
+    "    QueryString=f\"create database {ATHENA_DB}\",\n",
+    "    ResultConfiguration={\"OutputLocation\": ATHENA_URL + \"/queries/\"},\n",
+    ")\n",
+    "stat(db_create, \"db_create\")\n",
+    "print(ATHENA_URL)\n",
+    "# print(ARN_DATABASE)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "753c6d28-c415-4d4e-9846-43c0fd6c4057",
+   "metadata": {},
+   "source": [
+    "## II. Granting Access to Athena\n",
+    "\n",
+    "By default, Quilt runs with very conservative permissions that do not allow access to [Amazon Athena](https://docs.aws.amazon.com/athena/latest/ug/what-is.html). To enable Athena SQL queries by your Quilt users, you must:\n",
+    "\n",
+    "### A. Create a new Athena policy.\n",
+    "\n",
+    "The standard [AmazonAthenaFullAccess](https://console.aws.amazon.com/iam/home#/policies/arn:aws:iam::aws:policy/AmazonAthenaFullAccess) policy is more permissive than necessary.  For production usage, we recommend creating a policy limited to only the above Bucket:\n",
+    "<!--pytest-codeblocks:cont-->"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "a2f2b070-6bfa-4161-aaab-d71ca481dbcb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/iam.html#IAM.ServiceResource.create_policy\n",
+    "# https://docs.aws.amazon.com/athena/latest/ug/workgroups-access.html\n",
+    "\n",
+    "AthenaQuiltAccess = {\n",
+    "    \"Version\": \"2012-10-17\",\n",
+    "    \"Statement\": [\n",
+    "        {\n",
+    "            \"Sid\": \"GrantQuiltAthenaFullAccess\",\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": \"athena:ListWorkGroups\",\n",
+    "            \"Resource\": \"*\",\n",
+    "        },\n",
+    "        {\n",
+    "            \"Sid\": \"GrantQuiltAthenaAccess\",\n",
+    "            \"Effect\": \"Allow\",\n",
+    "            \"Action\": [\n",
+    "                \"athena:Create*\",\n",
+    "                \"athena:Get*\",\n",
+    "                \"athena:ListDatabases\",\n",
+    "                \"athena:ListNamedQueries\",\n",
+    "                \"athena:ListPreparedStatements\",\n",
+    "                \"athena:ListQueryExecutions\",\n",
+    "                \"athena:StartQueryExecution\",\n",
```

Comment:
As a new feature, I'm working on discarding to show workgroups with `Status === 'DISABLED'` and with no `OutputLocation`. For this I need to call `getWorkgroup`, which is disallowed now on staging. I don't know where to put it:to "GrantQuiltAthenaFullAccess" or "GrantQuiltAthenaAccess"

Reply:
Small tip: all frontend Athena calls are in https://github.com/quiltdata/quilt/blob/master/catalog/app/containers/Bucket/Queries/requests/athena.ts
You can find them with Ctrl-F "await athena"